### PR TITLE
New Leetcode Submission Review.

### PR DESCRIPTION
**Pull request by SongYeongEng**
Added a new Leet Code question on 0013-roman-to-integer.

### CODE REVIEW

**Comment by kooinam** [comment]
try to provides concise/clear title & descriptions

**Comment by kooinam** [comment]
https://www.linkedin.com/pulse/how-raise-really-good-pull-requestspr-rishi-prakash

**Comment by SongYeongEng** [comment]
Updated Title and Comment.

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Solution(object):
+    def uniqueOccurrences(self, arr):
+        """
+        :type arr: List[int]
+        :rtype: bool
+        """
+        
+        #how pust isnside dictionarry
+        
+        ihash = {}
```
Comment (kooinam) [inline 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py:10]: why use ihash?

Comment (kooinam) [reply 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py:10]: what does i stands for

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        """
+        
+        #Not yet learn space power
+        
+        l1 = []
+        l2 = []
+        l3 = []
+        
+        for num in nums1:
+            if num not in nums2:
```
Comment (kooinam) [inline 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py:16]: can we use single if operation here?

**Review by kooinam — APPROVED** [review]
looks good
1. there's space for optimisations, study more about memoisation
2.  be more conscious about variable namings
3. be more conscious/consistent about syntax

for example, 
`a=1` vs `a = 1`

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py (hunk 1 above)
Comment (SongYeongEng) [reply 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py:10]: cause sometimes I use keyword like hash, sum, list I saw it change color so am afraid calling into a function. I haven't taking care of writing clean code, optimization, big o notation.

**Review by SongYeongEng — COMMENTED** [review]
Reply to ihash

**Pull request merged**